tests/omp/test_omp_tridiag.f90

```diff
@@ -246,13 +244,13 @@ program test_omp_tridiag
 
    if (nrank == 0) then
       print'(a, f8.3, a)', 'Available BW:   ', deviceBW/2**30, &
-                           ' GiB/s (per NUMA zone on ARCHER2)'
+         ' GiB/s (per NUMA zone on ARCHER2)'
```

Comment:
Is this also fprettify's auto fix?

Reply:
yes, it is also fprettify here. Though I don't mind that one too much

Reply:
It looks okay. But if you're bothered with the whitespace between `print` and `*` you can add
`whitespace-print = false`
to the `.fprettify.ini` file if you want.  But then I think you can't put a space even after comma, so it becomes `print*,'something'` if I remember right. There is really not enough control over minor fprettify settings unfortunately.

Reply:
I actually prefer `print *` over `print*`, so I don't mind the current syntax. I think I'd rather have `print '(xyz)'` versus `print'(xyz)'`, but I can live with that. As long as it is somewhat consistent and I don't have to manually tweak it I am fine with most syntax rule.